lionblon-hojun/src/compoents/Comment/index.jsx

```diff
@@ -0,0 +1,77 @@
+import { useState } from "react";
+import comments from "../../data/comments"; // dummy data
+import CommentElement from "./CommentElement";
+
+const Comment = ({ postId }) => {
+  const [commentsarr, setcommentsarr] = useState(comments);
+  const [commentInput, setcommentInput] = useState("");
```

Comment:
```suggestion
  const [commentsArray, setCommentsArray] = useState(comments);
  const [commentInput, setCommentInput] = useState("");
```
여기도 Camel Case 적용해주시고, 약어를 적기보다는 단어 전부를 적어주는 게 좋을 것 같아요!